app/index.html

```diff
@@ -55,14 +55,11 @@
                         <label for="selection-currency">Wybierz walutę:</label>
                     </div>
                     <div class="col-md-4">
-                        <select ng-model="item" id="selection-currency" ng-options="item for item in items" ng-change="changeCurrency()" class="form-control"></select>
+                        <select ng-model="item" id="selection-currency" ng-options="item for item in items" ng-change="changeCurrency()"
+                                class="form-control"></select>
```

Comment:
I think it will be better if you use "names" for *item* in *ng-options*. Now I have to guess what kind of items you want to displays with select.